src/theming.jl

```diff
@@ -47,7 +47,7 @@ const minimal_default = Attributes(
     padding = Vec3f0(0.05),
     raw = false,
     SSAO = Attributes(
-        enable = Node(false),
+        # enable = Node(false),
         bias = Node(0.025f0),       # z threshhold for occlusion
```

Comment:
These should be automatically converted to nodes, so you shouldn't have explicitly declare it.

Reply:
fixed it